### ISSUE TITLE
bug(test): Flakey test, again

### ISSUE THREAD

**Issue by gaocegege** [tensorchord/envd#365]
https://github.com/tensorchord/envd/pull/364

We got some flakey tests

- https://github.com/tensorchord/envd/runs/6911493545?check_suite_focus=true
- https://github.com/tensorchord/envd/runs/6911398703?check_suite_focus=true

**Comment by gaocegege** [comment]
#495 fixes one problem.

**Comment by gaocegege** [comment]
Cannot reproduce it.

**Comment by gaocegege** [comment]
```
Will run 2 of 2 specs
------------------------------
• [FAILED] [0.002 seconds]
home manager
/home/runner/work/envd/envd/go/src/github.com/tensorchord/envd/pkg/home/manager_test.go:28
  when initialized
/home/runner/work/envd/envd/go/src/github.com/tensorchord/envd/pkg/home/manager_test.go:29
[It] should initialized successfully
/home/runner/work/envd/envd/go/src/github.com/tensorchord/envd/pkg/home/manager_test.go:30
Expected
      <types.BuilderType>: tcp
  to equal
      <types.BuilderType>: docker-container
In [It] at: /home/runner/work/envd/envd/go/src/github.com/tensorchord/envd/pkg/home/manager_test.go:38
------------------------------
•
Ran 2 of 2 Specs in 0.007 seconds
FAIL! -- 1 Passed | 1 Failed | 0 Pending | 0 Skipped
```

**Comment by kemingy** [comment]
e2e test failed 11 times: https://github.com/tensorchord/envd/runs/7460599803?check_suite_focus=true

Can check the log for details.

**Comment by gaocegege** [comment]
Closed in #658
